websites/T/Twitch/presence.ts

```diff
@@ -420,7 +420,8 @@ presence.on("UpdateData", async () => {
 
 			if (path.includes("/moderator/")) {
 				presenceData.details = strings.modStreamer;
-				presenceData.state = getElement(".stream-info-card p > a");
+				presenceData.state =
+					path.match(/moderator\/([a-z\d][\w]{2,24})$/i)?.[1] ?? "Not selected";
```

Comment:
There is one more method of getting which streamer it is.
```suggestion
					path.match(/moderator\/([a-z\d][\w]{2,24})$/i)?.[1] ?? document.querySelector('[data-a-target="player-info-title"]')?.textContent ?? "Not selected";
```
![image](https://github.com/PreMiD/Presences/assets/42322979/8335332a-08d0-40a9-b5b1-845238881a2a)